apps/package.json

```diff
@@ -55,6 +55,7 @@
     "chai-subset": "1.2.0",
     "chalk": "^1.1.3",
     "chrome-serialport": "bcjordan/chrome-serialport#2",
+    "classnames": "^2.2.5",
```

Comment:
Unclear to me whether linking to the repo here was meant as a sign of approval/disapproval/other?